modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -49,25 +47,67 @@ definition {
 	@description = "LPS-164617. Assert that it's possible to make changes in the FDS "
 	@priority = "4"
 	test CanBeEditted {
-
-		// To-do CanBeEditted Autom. test
-
 	}
 
 	@description = "LPS-164616. Assert that it's possible to create A new custom view and selected as current view"
 	@priority = "4"
 	test CanBeSaved {
-
-		// To-do CanBeSaved Autom. test
-
 	}
 
 	@description = "LPS-164618. Assert that it's possible can edit a new instance was previously created"
 	@priority = "4"
 	test CanCreateMultiple {
+		task ("And Given A custom view is created and selected") {
+			FDSCustomView.createNewView(key_nameView = "FrontendDataSet View");
+
+			FDSCustomView.assertCustomView(key_nameCustomView = "FrontendDataSet View");
```

Comment:
Remove this assertion. If you want you can try using VerifyElementPresent.

Reply:
Corrected.